Newsletters: add 360 (2025-06-27)

harding opened the pull request:
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Stack Exchange @bitschmidty 
- [x] Notable merges @Gustavojfe

bitschmidty commented on _posts/en/newsletters/2025-06-27-newsletter.md:
```diff
@@ -0,0 +1,118 @@
+
+- [LND #9950][] adds a new `include_auth_proof` flag to the `DescribeGraph`, `GetNodeInfo` and `GetChanInfo` RPCs and to their corresponding `lncli` commands. Including this flag returns the [channel announcement][topic channel announcements] signatures.
+
+- [LDK #3868][] reduces the precision of the [HTLC][topic htlcs] hold time for attributable failure (see Newsletter [#349][news349 attributable]) payloads from 1-millisecond to 100-millisecond units, to mitigate timing-fingerprint leaks. This aligns LDK with the latest updates to the [BOLTs #1044][] draft.
```
```suggestion
- [LDK #3868][] reduces the precision of the [HTLC][topic htlc] hold time for attributable failure (see Newsletter [#349][news349 attributable]) payloads from 1-millisecond to 100-millisecond units, to mitigate timing-fingerprint leaks. This aligns LDK with the latest updates to the [BOLTs #1044][] draft.
```

bitschmidty commented on _posts/en/newsletters/2025-06-27-newsletter.md:
```diff
@@ -0,0 +1,118 @@
+[news310 ipc]: /en/newsletters/2024/07/05/#bitcoin-core-30200
+[news356 failures]: /en/newsletters/2025/05/30/#eclair-3065
+[news349 attributable]: /en/newsletters/2025/04/11/#ldk-2256
+[news359 eclair]: en/newsletters/2025/06/20/#eclair-3110
```
```suggestion
[news359 eclair]: /en/newsletters/2025/06/20/#eclair-3110
```

bitschmidty commented on _posts/en/newsletters/2025-06-27-newsletter.md:
```diff
@@ -0,0 +1,118 @@
+
+- [Libsecp256k1 #1678][] cmake: add a helper for linking into static libs
+
+- [BIPs #1803][] clarifies [BIP380][]’s [descriptor][topic descriptors] grammar by allowing all common hardened-path markers, while [#1871][bips #1871], [#1867][bips #1867], and [#1866][bips #1866] refine [BIP390][]’s [MuSig2][topic musig] descriptors by tightening key path rules, permitting repeated participant keys, and explicitly restricting multipath child derivations.
```
maybe

```suggestion
- [BIPs #1803][] clarifies [BIP380][]’s [descriptor][topic descriptors] grammar by allowing all common hardened-path markers, while [#1871][bips #1871], [#1867][bips #1867], and [#1866][bips #1866] refine [BIP390][]’s [MuSig2][topic musig] descriptors by tightening key path rules, permitting repeated participant keys, and explicitly restricting multi-path child derivations.
```

harding commented:
Made edits for all feedback (thanks @bitschmidty @LarryRuane !), added lede, releases/RCs, and topic entries.  Fixed my broken links (sorry!).  Reviewed sections by @bitschmidty and @Gustavojfe (thanks!).  Thanks everyone!

LarryRuane commented on _posts/en/newsletters/2025-06-27-newsletter.md:
```diff
@@ -0,0 +1,159 @@
+  propagation.
+
+- [Why is attacker revenue in selfish mining disproportional to its hash-power?]({{bse}}53030)
+  Antoine Poinsot follows up to this and [another]({{bse}}125682) older [selfish
```
```suggestion
  Antoine Poinsot follows up on this and [another]({{bse}}125682) older [selfish
```

LarryRuane commented on _posts/en/newsletters/2025-06-27-newsletter.md:
```diff
@@ -0,0 +1,159 @@
+
+- [Why is attacker revenue in selfish mining disproportional to its hash-power?]({{bse}}53030)
+  Antoine Poinsot follows up to this and [another]({{bse}}125682) older [selfish
+  mining][topic selfish mining] question, pointing out "The difficulty
```
```suggestion
  mining][topic selfish mining] question, pointing out, "The difficulty
```

LarryRuane commented on _posts/en/newsletters/2025-06-27-newsletter.md:
```diff
@@ -0,0 +1,159 @@
+  exposes all the library’s object files to allow parent projects, such as
+  Bitcoin Core’s planned [libbitcoinkernel][libbitcoinkernel project], to link
+  those objects directly into their own static libraries. This solves the
+  problem of CMake lacking a native mechanism to link static libraries into
+  another and spares downstream users from providing their own `libsecp256k1`
```
```suggestion
  problem of CMake lacking a native mechanism to link static libraries together
  and spares downstream users from providing their own `libsecp256k1`
```
I'm not sure this suggestion is correct, but something seems wrong with the way it's written currently.

bitschmidty replied:
Nothing is raising any flags for me on this PR summary, so Im going to publish as is and we can correct it later if something is off

LarryRuane commented on _posts/en/newsletters/2025-06-27-newsletter.md:
```diff
@@ -0,0 +1,159 @@
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Is there any way to block Bitcoin Knots nodes as my peers?]({{bse}}127456)
+  Vojtěch Strnad provides an approach to blocking peers based on user agent strings
```
```suggestion
  Vojtěch Strnad provides an approach to blocking peers based on user-agent strings
```
